scripts/gitlab/lingua.dic

```diff
@@ -195,6 +195,7 @@ SAFT
 scalability
 SDF
 sending/S
+ServiceFactory
```

Comment:
I don't see `ServiceFactory` introduced here, do you run the same command for spellcheck as CI does?
https://github.com/paritytech/polkadot/blob/34ad3ab3ba768de1637a491df26c9e384288394e/.gitlab-ci.yml#L175

Reply:
It was the polkadot CI that was complaining about it. But have been told to ignore this polkadot CI so... I can remove it if you want?

Reply:
Ah, that's because CI only checks changed files. Then let's leave and merge it as is and when @drahnr is back from vacation, hopefully he will confirm this is the right fix.